test/hotspot/jtreg/runtime/cds/appcds/SharedArchiveConsistency.java

```diff
@@ -374,13 +210,8 @@ public static void main(String... args) throws Exception {
             throw new RuntimeException("Arg must be 'on' or 'auto'");
         }
         shareAuto = args[0].equals("auto");
-
-        // must call to get offset info first!!!
-        getFileOffsetInfo();
-        Path currentRelativePath = Paths.get("");
-        String currentDir = currentRelativePath.toAbsolutePath().toString();
-        System.out.println("Current relative path is: " + currentDir);
-        // get jar file
+        WhiteBox box = WhiteBox.getWhiteBox();
+        CDSArchiveUtils.initialize(box);  // all offsets available
```

Comment:
After this refactoring, I don't think WhiteBox is required in this file.
So I think CDSArchiveUtils could create a WhiteBox instance instead of requiring the caller to supply one.

Reply:
The util class is just a utility class, I have no idea if we can do this, let me try --- it might work.